calculator.js

```diff
@@ -1,28 +1,56 @@
+const calculateUserInput = function (error, promptInput) {
+  //   console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
 
-const exampleAdditionInput = {
-  num1: 3,
-  num2: 5,
-  operation: 'add',
-}
+  let result;
 
-const calculateUserInput = function (error, promptInput) {
-  console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
-
-  // Questions to ask and answer:
-  // What is promptInput?
-  // What data type? What does it hold? What does it represent?
-  // How do we read values from it? What syntax?
-  // How can we use it?
-  // Can we call our existing functions now, inside of this function?
-}
+  //user input
+  const num1 = parseFloat(promptInput.num1);
+  const num2 = parseFloat(promptInput.num2);
+  const operator = promptInput.operation;
+
+  switch (operator.toLowerCase()) {
+    case '+':
+    case 'add':
+      result = num1 + num2;
+      console.log(`${num1} + ${num2} = ${result}`);
+      break;
 
-// Example manual testing of calculator.  
-// calculateUserInput({}, {
-//   num1: 3,
-//   num2: 4,
-//   operation: 'add',
-// });
+    case '-':
+    case 'subtract':
+      result = num1 - num2;
+      console.log(`${num1} - ${num2} = ${result}`);
+      break;
+
+    case '*':
+    case 'multiply':
```

Comment:
Great use of the switch statement here! by convention, most programmers don't put a linebreak in between different cases, as it helps keep more of the code (and therefore more of the context for the switch statement) on the same screen.
```suggestion
    case '-':
    case 'subtract':
      result = num1 - num2;
      console.log(`${num1} - ${num2} = ${result}`);
      break;
    case '*':
    case 'multiply':
```